app/src/org/commcare/android/adapters/SquareButtonViewHolder.java

```diff
@@ -0,0 +1,30 @@
+package org.commcare.android.adapters;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import org.commcare.android.view.SquareImageView;
+import org.commcare.dalvik.R;
+
+/**
+ * Holds views for a home screen button
+ *
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+public class SquareButtonViewHolder extends RecyclerView.ViewHolder {
+    public final SquareImageView imageView;
+    public final RelativeLayout cardView;
+    public final TextView textView;
+    public final TextView subTextView;
+
+    public SquareButtonViewHolder(View view) {
```

Comment:
I think this is the standard pattern. ViewHolders hold info and shouldn't do anything else, so I think view inflation shouldn't occur in this class. Plus the `parent` param from the adapter's `onCreateViewHolder` _might_ be important